src/app/services/models/annotation.service.ts

```diff
@@ -62,3 +66,62 @@ export class AnnotationService {
     );
   }
 }
+
+interface ResolverNames {
```

Comment:
custom resolvers are pretty rare right? maybe document why this one is needed (virtual service that need multiple other services)